src/ENVPAK.CPP

```diff
@@ -7,9 +7,16 @@
 /*------------------------------- INCLUDES --------------------------------*/
 #include "cnglob.h"
 // #include "cse.h"
+#if !defined( ELIMINATE_WINDOWS)
+#include <windows.h>
+#include <direct.h>	// getcwd
```

Comment:
These will eventually need to be wrapped in an `#if (_WIN32)`.

Reply:
The `direct.h` can go but yes. Eventually, we can have 
```
#if (_WIN32)
#include <windows.h>
#else 
#include <unistd.h>
#endif
```
Since they are both used for `enExePath`.